### PR TITLE
Updated codebase to Python3.6

### PR DESCRIPTION
**Pull request by TheDataLeek**
Major compatibility changes:

* Complete route rewrite
    * Simplified structure
    * Removed blog capabilities (not needed)
* Removed HAML, replaced with HTML and Jinja2
* SCSS is not longer compiled at deploytime, but rather dynamically per request (also cached)
    * TODO: let's talk about how we want to handle this
* Updated to GAE Flex environment
* Using `gunicorn` for prod

### CODE REVIEW

**Review by svvitale — DISMISSED** [review]

File: run.py [hunk 1]
```diff
@@ -1,21 +1,83 @@
...
+
+def main(args):
+    configure(os.getenv('FLASK_CONFIG') or 'default')
+
+    if args.test is True:
+        pid = os.fork()
+        if pid == 0:
+            run_server(mode='debug')
+        else:
+            os.system(f'py.test')
```
Comment (svvitale) [inline run.py]: Is there a way to run this directly without making a system call?

File: ghostdriver.log [hunk 2]
```diff
@@ -1,9 +1,5 @@
...
-[INFO  - 2017-10-23T20:00:53.590Z] SessionManagerReqHand - _postNewSessionCommand - New Session Created: e0187320-b82c-11e7-baef-9f397319b06c
-[INFO  - 2017-10-23T20:05:52.775Z] SessionManagerReqHand - _cleanupWindowlessSessions - Asynchronous Sessions clean-up phase starting NOW
-[INFO  - 2017-10-23T20:05:52.775Z] SessionManagerReqHand - _cleanupWindowlessSessions - Deleted Session 'e0187320-b82c-11e7-baef-9f397319b06c', because windowless
-clean-up phase starting NOW
-[INFO  - 2017-10-23T20:05:05.771Z] SessionManagerReqHand - _cleanupWindowlessSessions - Deleted Session 'c424fe40-b82c-11e7-99c5-cdbaed70dee3', because windowless
+[INFO  - 2017-10-23T20:43:21.287Z] GhostDriver - Main - running on port 33385
```
Comment (svvitale) [inline ghostdriver.log]: Should this file be checked in?

Comment (TheDataLeek) [reply ghostdriver.log]: No, I'll delete.

File: run.py (hunk 1 above)
Comment (TheDataLeek) [reply run.py]: Erhm, subprocess is another option. Preferences?

I want to use shell though as we need the test script to be compatible on any device/platform.

Comment (svvitale) [reply run.py]: https://docs.pytest.org/en/latest/usage.html#calling-pytest-from-python-code

Comment (TheDataLeek) [reply run.py]: *nifty!* I'll implement.

File: .gitignore [hunk 3]
```diff
@@ -16,3 +9,164 @@ google-cloud-sdk/
...
 # ignore the credentials file for google app engine
 client-secret.json
+
+# Created by .ignore support plugin (hsz.mobi)
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and Webstorm
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+# User-specific stuff:
+.idea/**/workspace.xml
```
Comment (svvitale) [inline .gitignore]: I generally leave out all of `.idea/*`.

**Review by svvitale — APPROVED** [review]

**Pull request merged**